CONTRIBUTING.md

```diff
@@ -105,17 +103,22 @@ with what you'd like to do or have ideas that will help you do it.
 
 ### Adding a new library
 
-If you want to submit type stubs for a new library, you need to
+Stubs for new third party libraries are generally not accepted into
+typeshed anymore.  Such stubs should either be added directly to the
+third party package or distributed as a separate package.  See
+[PEP 561](https://www.python.org/dev/peps/pep-0561/) for details.
+
+When creating a separate package, you should
```

Comment:
Since we can't really control what happens outside of typeshed, I have softened the "need to" from the original text to a "should" here.